projects/packages/my-jetpack/_inc/style.scss

```diff
@@ -0,0 +1,7 @@
+@import '@automattic/jetpack-base-styles/style';
```

Comment:
👍 let's remove it from here.

Reply:
I think we could remove from `MyJetpackScreen` and keep it here since the other pages are not getting the `base-styles` if we keep it only on `MyJetpackScreen`.